Dry up the layouts and styles

csexton opened the pull request:
Originally the "default" layout used a separate layout and style sheet. This refactors all the subpages (info, faq, prospectus, Sponsorship) to use a common "page" layout.

There are now two layout:
- `page`
- `base`

Base is the bare layout that has just the HTML header, javascript includes, and other chrome.

Page adds a small bit of structure so that markdown files will render nicely.

---

It might look something like this:

![image](https://cloud.githubusercontent.com/assets/16963/16162161/a8618370-349f-11e6-9ed0-b175de6ef6e3.png)

![image](https://cloud.githubusercontent.com/assets/16963/16162157/a02da60c-349f-11e6-9dc7-f1ab2c397f62.png)

csexton committed:
Dry up the layouts and styles

Originally the "default" layout used a separate layout and style sheet.
This refactors all the subpages (info, faq, prospectus, Sponsorship) to
use a common "page" layout.

There are now two layout:

- `page`
- `base`

Base is the bare layout that has just the HTML header, javascript
includes, and other chrome.

Page adds a small bit of structure so that
markdown files will render nicely.

adunkman commented on _layouts/page.html:
```diff
@@ -0,0 +1,22 @@
+<section class="dark section">
+  <hr>
+  <div class="container">
+      <p>&copy 2016 - Ruby for Good</p>
```
Looks like you’re missing a semicolon here: `&copy;`

adunkman commented:
Pulled this down and ran it locally — looks like `coc.md` is still referencing the `default` layout, which no longer exists. 

One missing semicolon on `&copy;` above, but outside of that, everything else looks good! 

👍 with fixed `&copy;` and `coc.md` updated to the new layout name.

❤️ ❤️ ❤️

csexton committed:
Fix the copy html entity and generate the year

Now the year won't need to be edited every year. But the site will need
to be re-generated every year. Hopefully that won't be a problem.

csexton committed:
Use the "page" layout

So it renders something.

adunkman commented:
Look at those appropriate ©️s! 

![yyaaaas](https://dl.dropboxusercontent.com/u/424495/GIFs/happy-andrew.gif)
